tests/test_client.py

```diff
@@ -1,41 +1,100 @@
 import os
+import requests
 
 from geoapify.client import Client
 
 
 class TestClient:
     API_KEY = os.environ['GEOAPIFY_KEY']  # add yours to your environment variables
 
-    def test_place_details(self):
+    def test_place_details(self, monkeypatch):
+        class MockRequestsGet:
+            def __init__(self, url, params, headers):
+                pass
+
+            @staticmethod
+            def json():
+                return {'type': 'FeatureCollection', 'features': [{'type': 'Feature', 'properties': {'feature_type': 'details', 'description': 'Renovation pending - expected to be opened again by 2019', 'name': 'Hotel Astoria', 'wiki_and_media': {'wikidata': 'Q649690'}, 'categories': [], 'datasource': {'sourcename': 'openstreetmap', 'attribution': '© OpenStreetMap contributors', 'license': 'Open Database Licence', 'url': 'https://www.openstreetmap.org/copyright', 'raw': {'name': 'Hotel Astoria', 'osm_id': 4851793010, 'osm_type': 'n', 'wikidata': 'Q649690', 'addr:city': 'Bruxelles - Brussel', 'addr:street': 'Rue Royale - Koningsstraat', 'description': 'Renovation pending - expected to be opened again by 2019', 'addr:country': 'BE', 'addr:postcode': 1000, 'disused:tourism': 'hotel', 'addr:housenumber': 103}}, 'housenumber': '103', 'street': 'Rue Royale - Koningsstraat', 'city': 'City of Brussels', 'county': 'Brussels-Capital', 'state': 'Brussels-Capital', 'postcode': '1000', 'country': 'Belgium', 'country_code': 'be', 'formatted': 'Hotel Astoria, Rue Royale - Koningsstraat 103, 1000 City of Brussels, Belgium', 'address_line1': 'Hotel Astoria', 'address_line2': 'Rue Royale - Koningsstraat 103, 1000 City of Brussels, Belgium', 'lat': 50.8512746, 'lon': 4.3649087, 'place_id': '517f1a52a0aa7511405923e7e990f66c4940f00103f901727c30210100000092030d486f74656c204173746f726961'}, 'geometry': {'type': 'Point', 'coordinates': [4.3649087, 50.851274599]}}]}
```

Comment:
Note that such a long line violates PEP8. And it is ugly anyway adding such a lengthy dict into the test directly.

My suggestion: define those dictionaries at the bottom of test_client.py. And if you do so, use all upper case names and format them with e.g. https://codebeautify.org/python-formatter-beautifier: 

RES_PLACE_DETAILS = {...}